lost_pwd_page.php

```diff
@@ -65,7 +65,9 @@
 			</legend>
 			<ul id="login-links">
 				<li><a href="login_page.php"><?php echo lang_get( 'login_link' ); ?></a></li>
+				<?php if(  ( ON == config_get_global( 'allow_signup' ) )) { ?>
```

Comment:
The spacing around the brackets seems to be wrong two spaces on one side and no spaces on the other.  There also seem to be an extra set of ().